packages/venia-concept/src/components/MiniCart/kebab.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+import Icon from 'src/components/Icon';
+import classify from 'src/classify';
+import defaultClasses from './kebab.css';
+
+class Kebab extends Component {
+    render() {
+        const { classes, isOpen, onFocus, onBlur, children } = this.props;
```

Comment:
Rather than whitelisting DOM props such as `onFocus` and `onBlur` here, consider applying rest props, which don't have to be updated for each new use case.

```jsx
const { children, classes, isOpen, ...restProps } = this.props;

return (
  <div {...restProps} className={classes.root}>
    <ul>{children}</ul>
  </div>
);
```